packages/learn-card-core/src/wallet/plugins/ceramic/ceramic.ts

```diff
@@ -114,17 +159,24 @@ export const getCeramicPlugin = async <URI extends string = ''>(
                 if (!verificationResult.success) return undefined;
 
                 const streamId = verificationResult.data.split(':')[2];
-
-                return VCValidator.parseAsync(await readContentFromCeramic(streamId));
+                try {
+                    return VCValidator.parseAsync(await readContentFromCeramic(streamId));
```

Comment:
```suggestion
                    return await VCValidator.parseAsync(await readContentFromCeramic(streamId));
```

nit: This is that weird edge case where we have to `await` before returning or else the `try` won't catch the error!